### PR TITLE
Bn 1046 epoch data

### PR DESCRIPTION
**Pull request by nandotorterolo**
## Purpose
Clients (Annulus) may be interested in historical statistics about the blockchain.  Epochs are expected to be suitable lengths of time for aggregating data, so accessing EpochData may be useful.

## Pending PRs
- https://github.com/Topl/protobuf-specs/pull/70

## Testing
FetchEpochData

## Tickets
https://topl.atlassian.net/browse/BN-1046

### CODE REVIEW

File: blockchain/src/main/scala/co/topl/blockchain/interpreters/EpochDataInterpreter.scala [hunk 1]
```diff
@@ -165,9 +166,9 @@ object EpochDataEventSourcedState {
           startTimestamp = startTimestamp,
           endTimestamp = endTimestamp,
           transactionCount = 0,
-          totalTransactionReward = 0,
-          activeStake = activeStake,
-          inactiveStake = inactiveStake,
+          totalTransactionReward = Int128(ByteString.copyFrom(BigInt(0).toByteArray)),
```
Comment (o-aleksandrov-topl) [inline blockchain/src/main/scala/co/topl/blockchain/interpreters/EpochDataInterpreter.scala:169]: maybe create some method `toInt128` wuth all stuff like `Int128(ByteString.copyFrom(BigInt(0).toByteArray))` ? Looks like it used in many places

**Review by o-aleksandrov-topl — APPROVED** [review]

File: blockchain/src/main/scala/co/topl/blockchain/interpreters/EpochDataInterpreter.scala (hunk 1 above)
Comment (SeanCheatham) [reply blockchain/src/main/scala/co/topl/blockchain/interpreters/EpochDataInterpreter.scala:169]: Adding `import co.topl.numerics.implicits._` should provide the auto-conversions from Int, Long, or BigInt.  That should allow for `totalTransactionReward = 0`

**Comment by nandotorterolo** [comment]
FetchEpochDataOptional

File: project/Dependencies.scala [hunk 2]
```diff
@@ -10,7 +10,7 @@ object Dependencies {
   val fs2Version = "3.7.0"
   val logback = "1.4.7"
   val orientDbVersion = "3.2.19"
-  val protobufSpecsVersion = "ae3f01df" // scala-steward:off
+  val protobufSpecsVersion = "a2e916035fbe81a8a647aaec81cdc4dc24661d6f" // scala-steward:off TODO replace with main version
```
Comment (nandotorterolo) [inline project/Dependencies.scala:13]: TODO

**Review by SeanCheatham — APPROVED** [review]

**Comment by nandotorterolo** [comment]
Thanks for reviewing @SeanCheatham @o-aleksandrov-topl

**Pull request merged**